packages/@storefront/flux-capacitor/src/core/reducers/data/past-purchases.ts

```diff
@@ -182,3 +184,27 @@ export const updatePastPurchaseTemplate = (state: State, { payload }: Actions.Re
     ...state,
     template: payload
   });
+
+// tslint:disable-next-line max-line-length
+export const receiveMorePastPurchaseRefinements = (state: State, { payload: { navigationId, refinements, selected } }: Actions.ReceiveMorePastPurchaseRefinements) => {
```

Comment:
Instead of duplicating this code, import the navigation `receiveMoreRefinements` and use it directly, similar to line 145

Reply:
Implemented, resolving.